testgen/Cargo.toml

```diff
@@ -23,7 +23,7 @@ ed25519-dalek = { version = "1", default-features = false }
 gumdrop = { version = "0.8.0", default-features = false }
 simple-error = { version = "0.2.1", default-features = false }
 tempfile = { version = "3.1.0", default-features = false }
-chrono = { version = "0.4", default-features = false, features = ["clock"] }
+timelib = { package = "time", version = "0.3", default-features = false, features = ["std"] }
```

Comment:
Why the `timelib` alias?

Reply:
It was ambiguous with the top-level `time` module, but I've found a better way to disambiguate.